simulation/bombard.py

```diff
@@ -13,11 +13,11 @@
     print("python3 ./bombard.py <UUID> <AUTH_TOKEN>")
     sys.exit()
 
-num = 10
+num = 50
```

Comment:
should `num` be renamed to `iterations`, or `messages`?

Reply:
done. renamed to `number_of_requests_per_type`